frontend/src/components/AddSpot.tsx

```diff
@@ -49,10 +42,93 @@ type AddSpotProps = {
     handleSave(newSpot: Spot): void
 }
 
+interface TextFieldInput {
```

Comment:
Gibt es einen Grund warum du zu Interface gewechselt bist?
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Reply:
In den docs haben die mit interfaces gearbeitet :D Ist jetzt ein typ